roles/acm_spoke_mgmt/README.md

```
@@ -93,7 +94,8 @@ Name                         | Type   | Required | Default
 ---------------------------- | ------ | -------- | -------------------------------------------------- | ------------------------------------------------------------
 asm_source_repo              | string | yes      | -                                                  | GitOps repository that was used to deploy the ZTP cluster
 asm_target_revision          | string | yes      | -                                                  | Branch used to deploy the ZTP cluster
-asm_delete_ztp_resources     | boolean| yes      | true                                               | Deletes the ArgoCD applications and all the related cluster deployments resources
+asm_delete_ztp_resources     | boolean| no       | true                                               | Deletes the ArgoCD applications and all the related cluster deployments resources
+asm_cluster_name             | string | no       | If asm_delete_ztp_resources is set to true, name of the Managed Cluster the role must wait to be destroyed after removing the GitOps applications.
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 5; Actual: 4; Too few cells, row will be missing data

(MD056, table-column-count)